e2e/terraform/packer/ubuntu-bionic-amd64/provision.sh

```diff
@@ -2,7 +2,7 @@
 
 set -o errexit
 set -o nounset
-set +x
+set +ex
```

Comment:
This `+e` should be removed, we've got `set -o errexit` above.

Reply:
I somehow wasn't seeing the error I was expecting to see when running this without `set +e` but will remove